lib/maths/CBoostedTreeImpl.cc

```diff
@@ -444,6 +445,11 @@ void CBoostedTreeImpl::train(core::CDataFrame& frame,
         LOG_TRACE(<< "Test loss = " << m_BestForestTestLoss);
 
     } else if (m_CurrentRound < m_NumberRounds || m_BestForest.empty()) {
+        using TUIntMeanVarAccumulator =
```

Comment:
nit: this should be renamed to something like TFloatMeanVarAccumulator. (I'd also be inclined to just go with double: there is no real reason to save memory or runtime here.)